src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const value = 40;
+  const minimumDiscount = 20;
+  const maximumDiscount = 50;
+  const basePrice = days * value;
+
+  if (days <= 2) {
+    return basePrice;
+  } else if (days >= 3 && days <= 6) {
+    return basePrice - minimumDiscount;
+  } else if (days >= 7) {
+    return basePrice - maximumDiscount;
+  }
```

Comment:
Avoid using 'else if' after a return statement. You can remove the 'else' keyword to simplify the control flow.